pkg/apisix/resource.go

```diff
@@ -73,17 +73,18 @@ type item struct {
 }
 
 type routeItem struct {
-	UpstreamId string                 `json:"upstream_id"`
-	ServiceId  string                 `json:"service_id"`
-	Host       string                 `json:"host"`
-	Hosts      []string               `json:"hosts"`
-	URI        string                 `json:"uri"`
-	Vars       [][]v1.StringOrSlice   `json:"vars"`
-	Uris       []string               `json:"uris"`
-	Desc       string                 `json:"desc"`
-	Methods    []string               `json:"methods"`
-	Priority   int                    `json:"priority"`
-	Plugins    map[string]interface{} `json:"plugins"`
+	UpstreamId  string                 `json:"upstream_id"`
```

Comment:
Generally I like some kind of order in lists, arrays or structures but not sure how this affects the whole system.

So with ordering some people chose an alphabetically ordered list, and others use a custom sort order for maybe the first few items in the list and then alphabetically for the rest of the items in the list.  There are many ways to sort and design data structures. 

Ordering also makes it easier to find existing items in the lists and speeds up adding new items.  
So in this file  there seems to be two `lists` and some other files in this PR have `lists`.
A general code style comment that I always push.

Reply:
The order of fields affects the memory alignment but is not so important here, if that's really vital, we can optimize it in the future.